samples/CloudClipboard/CloudClipboard/CloudClipboard/wwwroot/lib/jquery-validation-unobtrusive/.bower.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "jquery-validation-unobtrusive",
```

Comment:
Bower is on life support these days;  We shouldn't be using it.

Reply:
I think we should be using it, so i did not change it. Do you agree?

Reply:
Ted's point of it being part of the template is salient here as well;  the template is in the wrong for including a deprecated library - but we should probably leave it as-is so that the sample matches what the template would generate for customers.

Reply:
Got it, Thanks for your reviewing